WORKSPACE

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 The StableHLO Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Workspace for StableHLO."""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+SKYLIB_VERSION = "1.3.0"
+
+http_archive(
+    name = "bazel_skylib",
+    sha256 = "74d544d96f4a5bb630d465ca8bbcfe231e3594e5aae57e1edbf17a6eb3ca2506",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/{version}/bazel-skylib-{version}.tar.gz".format(version = SKYLIB_VERSION),
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/{version}/bazel-skylib-{version}.tar.gz".format(version = SKYLIB_VERSION),
+    ],
+)
+
+LLVM_COMMIT = "02988fce76d8a95e2bc33e80b581c3b0b2c92755"
```

Comment:
Can this be fetched from `build_tools/llvm_version.txt`?

Reply:
Looks like WORKSPACE files can't read files, one option is moving `llvm_version.txt` to `llvm_version.bzl` and giving it a variable name to load. (I think) MLIR-HLO accomplishes this by automatically updating that commit / sha value in place when LLVM version changes.

Reply:
Another option is to run the script I just pushed via github actions, which will then trigger the bazel build. Command to run the script: `sh ./build_tools/github_actions/ci_build_bazel.sh .`

Reply:
This is an interesting idea, but I think that we'll want the Bazel build to be usable outside our CI as well, i.e. with standard Bazel, so that the repo can be built by humans both via CMake and Bazel.

Reply:
The LLVM_COMMIT can be fetched from a file like Kevin mentioned, but there isn't a way to compute the sha256 of the repo (or run a shell script in general) in the WORKSPACE through Bazel that I'm aware of, which is why I created a bash script to replace the string in the WORKSPACE file instead. I don't think adding the script as a target is also going to work since it is modifying the WORKSPACE file in place that Bazel depends on.